cli/src/pcluster/models/cluster.py

```diff
@@ -879,7 +878,9 @@ def export_logs(
                 stack_events_file = os.path.join(root_archive_dir, self._stack_events_stream_name)
                 export_stack_events(self.stack_name, stack_events_file)
 
-                create_logs_archive(root_archive_dir, output)
+                archive_path = create_logs_archive(root_archive_dir)
+                s3_path = upload_archive(bucket, bucket_prefix, archive_path)
```

Comment:
I thought the flow would be that the customer provides the link and thus doesn’t need to send large files over email — they already have the ability to see the logs with other commands so I had assumed this was for external debugging— open to either option though

Reply:
I was also on the fence about allowing the keep-s3 parameter because I generally think it would be good to have the flow be simple without many options